queries/mainnet_citizens.sql

```diff
@@ -0,0 +1,65 @@
+with
+-- For a permanent version of this query please visit: https://dune.xyz/queries/453009
+investment_claims as (
```

Comment:
In order to reduce code redundancy, could we use this sub_query also for gnosis chain? 
Could we somehow save the subqueries in their own text file, and then build the whole query in python with the sub_queries?

I know on gnosis chain there is no claim type 3, and the tokens are different. But maybe we can start thinking already about merging subqueries in order to reduce code redundancy.

I am not really sure that it's really valuable, and sometimes it's just easier to keep it with two queries. What do you think?

Reply:
I could rework this to remove redundancy! I had already started thinking about it and am now prepared to make the change. Note that it doesn't matter there is no USDC investment on gnosis chain because there simply would be no results for it so we don't need any special case for the code. I'll report back here when I have made the adjustments.

Reply:
Done!